frontend/src/app/pages/exercises/exercise/shared/exercise-map/shared/map-image-popup/map-image-popup.component.html

```diff
@@ -52,6 +52,15 @@ <h5 class="popover-header">
                     [ngModelInput]="heightInput"
                 ></app-display-validation>
             </div>
+            <div class="form-check form-switch pb-3">
+                <input
+                    type="checkbox"
+                    class="form-check-input"
+                    [ngModel]="mapImage.isLocked"
+                    (ngModelChange)="setLocked($event)"
+                />
+                <label class="form-check-label">Position sperren</label>
```

Comment:
Maybe put the label in front of the toggle? I am thinking of e.g. settings in iOS or Android (first the name and then the toggle).

Reply:
We've used a layout consistent to the existing "Verbindungen anzeigen" toggle in the sidebar of the editor.

I don't have a preference whether the toggle is on the left or the right of the label, but feel free to open a separate issue to discuss this.

By the way: At the moment, labels and input element have no logical connection. I've created #552 for this.

Reply:
Ok, doing this later and making it consistent in the UI.